### PR TITLE
update for 2.0

### PR DESCRIPTION
**Pull request by jsanford8**

### CODE REVIEW

File: source/index.html.md [hunk 1]
```diff
@@ -50,32 +51,33 @@ mux.monitor('#my-video', {
...
     video_series: '', // ex: 'Weekly Great Videos'
     video_producer: '', // ex: 'Bob the Producer'
     video_content_type: '', // 'short', 'movie', 'episode', 'clip', 'trailer', or 'event'
-    video_language: '', // ex: 'en'
+    video_language_code: '', // ex: 'en'
     video_variant_name: '', // ex: 'Spanish Hard Subs'
     video_variant_id: '', // ex: 'abcd1234'
-    video_duration: '', // in seconds, ex: 120.0
+    video_duration: '', // in milliseconds, ex: 120000
     video_stream_type: '', // 'live' or 'on-demand'
```
Comment (heff) [inline source/index.html.md]: We should probably remove `video_stream_type` until we know exactly how we want to use it.

File: source/index.html.md [hunk 2]
```diff
@@ -161,35 +164,37 @@ video_title | example: 'Awesome Show Episode 1'
...
 var myVideo = document.querySelector('#my-video');
 myVideo.src = 'nextVideo.mp4';
 
-mux.monitor('#my-video').changeVideo({
+mux.emit('#my-video', 'videochange', {
   video_id: 'abc345',
   video_title: 'My Other Great Video',
-  video_series: 'Weekly Great Videos'
+  video_series: 'Weekly Great Videos',
+  ...
```
Comment (heff) [inline source/index.html.md:187]: I worry about these ellipsis for the copy/pasters. Think we're safe enough?

File: source/index.html.md [hunk 3]
```diff
@@ -221,3 +226,9 @@ When `changeVideo` is called it removes all previous video data and resets all m
 <br>
 <br>
 <br>
+<br>
```
Comment (heff) [inline source/index.html.md:230]: wewt!

**Comment by heff** [comment]
lgtm!

File: source/index.html.md (hunk 2 above)
Comment (jsanford8) [inline source/index.html.md:187]: That's a good point I hadn't thought about, actually. Personally I would rather see the ... and have it break than forget that I'm supposed to put things in there, but I could be convinced to take them out.

Comment (jsanford8) [inline source/index.html.md:187]: That said, we have '...' in the video element example already (I didn't add that one), so copy-pasters might have problems anyway

**Pull request merged**